examples/stable_diffusion_xl/scripts/run_train_lora.sh

```diff
@@ -0,0 +1,25 @@
+export DEVICE_ID=$1
+export MS_PYNATIVE_GE=1
+# export MS_ASCEND_CHECK_OVERFLOW_MODE="INFNAN_MODE"
+# export MS_ASCEND_CHECK_OVERFLOW_MODE=1 # "INFNAN_MODE"
+export PYTHONPATH=$(pwd):$PYTHONPATH
```

Comment:
The training process does not require these two environmental variables

Reply:
fixed